src/Config.php

```diff
@@ -39,11 +36,10 @@ public function getRules() {
 			],
 			'indentation_type' => true,
 			'line_ending' => true,
-			'list_syntax' => [
-				'syntax' => 'short',
-			],
```

Comment:
was this intentional? It's this here, right:
```php
list($a, $b) = …;
[$a, $b] = …;
```

I don't think we want to go back and allow the long one?

Reply:
I didn't catch this but no, I would not go back. Could you send a PR to fix it? We can do a v3.0.1

Reply:
Or is this rule not supported anymore and "short" by default now?

Reply:
PR in #8

Reply:
I removed it, because it is declared as default in 3.2.

From https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.0.0/UPGRADE-v3.md

### Changed default values of options

Rule | Option | Old value | New value
---- | ---- | ---- | ----
`list_syntax` | `syntax` | `'long'` | `'short'`

Reply:
Fancy, sorry

Reply:
Good to know, that someone does a real review...